packages/katello/katello/katello-remove

```diff
@@ -44,10 +45,15 @@ CONFIG_FILES = [
   '/etc/puppetlabs',
   '/etc/qpid-dispatch',
   '/etc/qpid',
+  '/etc/smart_proxy_dynflow_core',
   '/etc/squid',
   '/etc/sudoers.d/foreman-proxy',
-  '/etc/sysconfig/foreman.rpmsave',
-  '/etc/sysconfig/puppetserver.rpmsave',
+  '/etc/systemd/system/multi-user.target.wants/pulp_celerybeat.service',
```

Comment:
Why are these needed? Doesn't the Pulp RPM removal clean this up? If not, then that's a bug in itself IMHO.

Reply:
What I meant was:
```suggestion
  # https://pulp.plan.io/issues/5195
  '/etc/systemd/system/multi-user.target.wants/pulp_celerybeat.service',
```